cds/webapi/C#/BlazorCDS/BlazorCDS/Program.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Net.Http;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Text;
+using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using BlazorCDS.Models;
+
+namespace BlazorCDS
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var builder = WebAssemblyHostBuilder.CreateDefault(args);
+            builder.RootComponents.Add<App>("app");
+
+            builder.Services.AddTransient(sp => new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) });
+
+            //Get configuration data about the Global Discovery API set in wwwroot/appsettings.json
+            var GDSWebApiConfig = builder.Configuration.GetSection("GDSWebAPI");
```

Comment:
```
//Get configuration data about the Global Discovery API set in wwwroot/appsettings.json
var GDSWebApiConfig = builder.Configuration.GetSection("GDSWebAPI");
var gdsResourceUrl = GDSWebApiConfig.GetSection("ResourceUrl").Value;
```
The value this is retrieving is a constant: `https://globaldisco.crm.dynamics.com`
I think this could skip storing and retrieving this from the config. Couldn't it just be hard-coded?

Reply:
set the value of gdsResourceUrl instead of getting from appSettings.json file